src/autogluon/bench/cloud/aws/batch_stack/lambdas/lambda_function.py

```diff
@@ -0,0 +1,186 @@
+import os
+import uuid
+from itertools import product
+
+import boto3
+import yaml
+
+aws_batch = boto3.client("batch")
+s3 = boto3.client("s3")
+
+
+def submit_batch_job(env: list, job_name: str, job_queue: str, job_definition: str):
+    container_overrides = {"environment": env}
+    response = aws_batch.submit_job(
+        jobName=job_name,
+        jobQueue=job_queue,
+        jobDefinition=job_definition,
+        containerOverrides=container_overrides,
+    )
+    print(response)
+
+
+def download_config(s3_path: str):
+    file_path = os.path.join("/tmp", s3_path.split("/")[-1])
```

Comment:
Is it correct to always save something to the tmp folder? I assume this is because this function will be used in lambda? Some docstring to clarify would be helpful

Reply:
Yes `/tmp` is the only directory that's writable for lambda functions.